tools/serverpod_cli/lib/src/config/config.dart

```diff
@@ -34,7 +34,21 @@ class GeneratorConfig {
     required List<String> relativeDartClientPackagePathParts,
     required this.modules,
     required this.extraClasses,
-  }) : _relativeDartClientPackagePathParts = relativeDartClientPackagePathParts;
+    required this.serializeEnumValuesAsStrings,
+  }) : _relativeDartClientPackagePathParts =
+            relativeDartClientPackagePathParts {
+    assert(_instance == null, 'GeneratorConfig should only be loaded once.');
+    _instance = this;
+  }
```

Comment:
Question: This is a good change but has this branch been tested with all different commands we support in the CLI? I think we for example in the `create-migration` command actually load the generator config multiple times.

This might be a change for a different PR where we fix all these issues and add this to prevent more cases from popping up.

Reply:
The question you raise is a direct result of this particular change being a very ugly hack! See my longer comment in the discussion. I would like to see this go away by entirely getting rid of serialization to integers.